plugin.video.aod/resources/language/English/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
This is not needed immediately ofcourse. Just something you can do for new addons already.